_includes/_trip-grid.html

```diff
@@ -4,94 +4,6 @@
             <div class="feature-cards">
                 <div id="ourHolder" class="row" style="display: flex; flex-wrap: wrap;">
 
-                    <div class="south-africa card push-ends col-xs-12 col-sm-4 col-md-3">
```

Comment:
@adamtgeorge @seancdavis This is unrelated to briantome.com, can you confirm this is good to go to production this morning?